relay-server/src/actors/outcome.rs

```diff
@@ -15,9 +15,11 @@ use relay_filter::FilterStatKey;
 use relay_general::protocol::EventId;
 
 use crate::actors::project::RateLimit;
-use crate::metrics::RelayCounters;
 use crate::ServerError;
 
+#[cfg(feature = "processing")]
+use crate::metrics::RelayCounters;
```

Comment:
_nit_: Move into the module `real_implementation`?

Reply:
good point